ekip/nationalparks/management/commands/field_trip.py

```diff
@@ -5,7 +5,14 @@
 from nationalparks.management.commands import ficorcleaner as fc
 from nationalparks.models import FieldTripSite, BestVisitTime, YouthFacility
 
-
+#Begin by cleaning house!
+if FieldTripSite.objects.exists():
```

Comment:
This won't quite work, because this is only checking against the rows in this table. 

Looks like the best route would be like:

`from django.db import connection
if FieldTripSite.objects.model._meta.db_table in connection.introspection.table_names():
    FieldTripSite.objects.all().delete()
if BestVisitTime.objects.model._meta.db_table in connection.introspection.table_names():
    BestVisitTime.objects.all().delete()
if YouthFacility.objects.model._meta.db_table in connection.introspection.table_names():
    YouthFacility.objects.all().delete()`